src/components/auth/Register.tsx

```diff
@@ -53,6 +61,7 @@ const Register: FC<{ setModalOpen: Dispatch<SetStateAction<AuthModals>>; }> = ({
                         name="name"
                         id="name"
                         placeholder="Your Name"
+                        required={true}
```

Comment:
Nu e necesar sa folosesti `={true}` deoarece daca adaugi doar `required` e de ajuns. O data pus in html, el este setat pe true.